api/api.go

```diff
@@ -178,6 +178,10 @@ type TLSConfig struct {
 	// the Nomad server SSL certificate.
 	CAPath string
 
+	// CAInMemCert is the PEM-encoded CA cert to use to verify the Nomad server
+	// SSL certificate.
+	CAInMemCert []byte
+
 	// ClientCert is the path to the certificate for Nomad communication
 	ClientCert string
```

Comment:
We probably should expose a raw field for `ClientCert` and `ClientKey` values.  I can follow up with that as well.

Reply:
Makes sense. I added both.